Assignment2-Python/stringCompression.py

```diff
@@ -0,0 +1,32 @@
+def compress_string(string):
+    if not string:
+        return ''
+
+    compress = []
+    count = 1
+
+    for i in range(1, len(string)):
+        if string[i] == string[i-1]:
+            count += 1
+        else: 
+            compress.append(string[i-1] + str(count))
+            count = 1
+    
+    compress.append(string[-1] + str(count))
+    compress_string = ''.join(compress)
+
+    if len(compress_string) < len(string):
+        return compress_string
+    else:
+        return string
```

Comment:
Because you're returning in both the `if` and `else` conditions with no extra computation, you can do this cleanly in one line using a ternary.
```suggestion
    return compress_string if len(compress_string) < len(string) else string
```